user/errors.rs

```diff
@@ -0,0 +1,13 @@
+use error::typedb_error;
+
+typedb_error!(
+    pub UserCreateError(component = "User create", prefix = "USC") {
+        Unexpected(1, "TODO: unexpected error"),
+    }
+);
+
+typedb_error!(
```

Comment:
`USD`? Check that there are no prefix collisions

Reply:
Just checked that `USC` is unique, there's no other error category that uses that three letters.

Reply:
Aah I get what you meant now. Fixed in 92cda1e30049f05fbbf2a59996039a7aac04012a.